src/main/java/com/aviva/CarRecommendations/CarRecommender.java

```diff
@@ -5,36 +5,64 @@
 import com.aviva.DataAccess.CSVCarData;
 import com.aviva.DataAccess.CarDataInterface;
 import com.aviva.Entities.AccountHolder;
-import com.aviva.Entities.Car;
+
 import org.json.JSONArray;
 import org.json.JSONObject;
 
 import java.io.IOException;
+import com.aviva.DataProcess.CarDataProcess;
 import java.util.ArrayList;
+import com.aviva.Entities.Car;
+import java.util.Random;
 
 public class CarRecommender {
-    public static JSONObject getRecommendedCars(AccountHolder user) throws IOException, InterruptedException, ClassNotFoundException {
-        float approvedLoanAmount = SensoRate.getApprovedLoanAmount(user);
-        CarDataInterface carData = new CSVCarData();
-        ArrayList<Car> availableCars = carData.loadCarData("../../../../../../data/Car_Data.csv");
-
-        JSONObject carsJSON = new JSONObject();
-        JSONArray recommendedCars = new JSONArray();
-        for (Car currentCar : availableCars) {
-            if (currentCar.getPrice() <= approvedLoanAmount) {
-                JSONObject carJSON = currentCar.toJSON();
-                recommendedCars.put(carJSON);
+//    public static JSONObject getRecommendedCars(AccountHolder user) throws IOException, InterruptedException, ClassNotFoundException {
```

Comment:
This is way too messy. Either delete the whole thing (if we really need old code we can use github to check it history), or only keep the differing lines as comments.